webpack.config.js

```diff
@@ -0,0 +1,28 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  entry: './src/index.js',
+
+  devServer: {
+    static: './dist',
+  },
+  plugins: [
+    new HtmlWebpackPlugin({
+      template: './src/index.html',
+    }),
+  ],
+  output: {
+    filename: '[name].bundle.js',
+    path: path.resolve(__dirname, 'dist'),
+    clean: true,
+  },
+  module: {
+    rules: [
+      {
+        test: /\.css$/i,
+        use: ['style-loader', 'css-loader'],
+      },
+    ],
+  },
+};
```

Comment:
- [ ] The 'mode' option has not been set in the `webpack.confog.js`, Webpack will fallback to 'production' for this value.
I recommend setting the 'mode' option to 'development'or 'production' to enable defaults for each environment.
Learn more: https://webpack.js.org/configuration/mode/